### PR TITLE
Task/predictive features cosmetic fixes

### PR DESCRIPTION
**Pull request by taoteg**
## Overview: ##

A quick PR to fix some cosmetic label issues on the predictive charts table and to cleanup some lingering CSS linting issues. 

## Related Jira tickets: ##

* No ticket, quick task.

## Testing Steps: ##

1. Run project, make sure everything looks the same.
2. See a new chart title for Predictive Features.

## UI Photos:

Screen Shot 2021-08-23 at 4 52 41 PM

### CODE REVIEW

**Comment by taoteg** [comment]
The last of my quick stuff for today.

**Comment by taoteg** [comment]
also - dunno where tat line came from, it doesn't render that way live. :P

File: client/setupTest.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // following are related to plotly and jest
 // https://stackoverflow.com/questions/48828759/unit-test-raises-error-because-of-getcontext-is-not-implemented
 // https://github.com/plotly/react-plotly.js/issues/115
-window.URL.createObjectURL = function() {};
+window.URL.createObjectURL = function createObjectURL() {};
```
Comment (nathanfranklin) [inline client/setupTest.js:4]: good fix 👍

File: client/src/components/ProTx/components/util.js [hunk 2]
```diff
@@ -769,6 +756,7 @@ export const getObservedFeaturesPlotData = (
     observedFeaturesPlotState: plotState,
     observedFeatureTargetValue: observedFeatureValue
   };
+  // console.log(observedFeaturesPlotData);
```
Comment (nathanfranklin) [inline client/src/components/ProTx/components/util.js:759]: drop

File: client/src/components/ProTx/components/charts/DebugPlot.js [hunk 3]
```diff
@@ -99,6 +99,7 @@ function DebugPlotRedux({
     }
 
     if (plotTypeDebug === 'observedFeatures') {
+      // console.log(plotDataDebug);
```
Comment (nathanfranklin) [inline client/src/components/ProTx/components/charts/DebugPlot.js:102]: drop

File: client/src/components/ProTx/components/charts/ReportChart.js [hunk 4]
```diff
@@ -41,11 +40,11 @@ function ReportChart({
           year={year}
           selectedGeographicFeature={selectedGeographicFeature}
           data={data}
+          // debug
```
Comment (nathanfranklin) [inline client/src/components/ProTx/components/charts/ReportChart.js:43]: drop 🤔

**Review by nathanfranklin — APPROVED** [review]
LGTM

Left some minor comments of things to drop but none are important

**Pull request merged**